packages/paste-core/components/ai-chat-log/src/utils.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+
+// Hook to animate text content of React elements
+export const useAnimatedText = (children: React.ReactNode, speed = 10): React.ReactNode => {
```

Comment:
question: why is `speed` a prop here? did you consider exposing speed as a prop set by the consumer?